rag_experiment_accelerator/doc_loader/tests/test_pdf_loader.py

```diff
@@ -8,15 +8,15 @@
 @pytest.mark.parametrize(
     "pypdf_enabled, expected_content",
     [
-        (True, "this is a sample pdf with text."),
+        (True, "col1  col2  col3  a b c 1 2 3        this is a sample text."),
         (
             False,
-            "this is a sample of text captured as an image and stored as pdf.this is a sample pdf with text.",
+            "col1 col2 col3 a b c 1 2 3this is a sample of text captured as an image and stored as pdf.this is a sample text.",
         ),
     ],
 )
 def test_load_pdf_files(pypdf_enabled: bool, expected_content: str):
-    file_path = "./data/pdf/text-as-image.pdf"
+    file_path = "./data/pdf/sample-pdf.pdf"
 
```

Comment:
**[Severity: Medium]** Consider using constants or configuration settings for file paths instead of hardcoding them. This will improve maintainability and adaptability of the code if path changes are required in the future.


---

**[peer-review-pal]** We are sorry for the intrusion. Feel free to ignore or delete the comments/issue with the tag - they are automatically generated by our [Hack project](https://hackbox.microsoft.com/hackathons/hackathon2024/project/81776) - repository **[Peer Review Pal](https://github.com/microsoft/peer-review-pal)**.
If you find them useful, want us to delete them, have any feedback at all or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding **#prp-ignore** to your PR description.